.circleci/config.yml

```diff
@@ -24,25 +25,16 @@ jobs:
       tag: '1.21'
     steps:
       - checkout
-      - run:  golangci-lint run superchain/... validation/... add-chain/...
+      - run:  make lint-all
   golang-modules-tidy:
     executor:
       name: go/default  # is based on cimg/go
       tag: '1.21'
     steps:
       - checkout
       - run:
-          name: tidy superchain module
-          command: go mod tidy
-          working_directory: superchain
-      - run:
-          name: tidy validation module
-          command: go mod tidy
-          working_directory: validation
-      - run:
-          name: tidy add-chain module
-          command: go mod tidy
-          working_directory: add-chain
+          name: tidy
+          command: make tidy-all
```

Comment:
Do you need to do `cp .env.example .env` before running the make commands in CI?

Reply:
Very likely, yes!